openapi/openapi.yml

```diff
@@ -0,0 +1,528 @@
+        get:
+            summary: Returns information about the node with the given main account.
+            tags:
+                - node
+            operationId: getNode
+            parameters:
+                - name: publicKey
+                  required: true
+                  description: The node's main account's public key.
+                  in: path
+                  schema:
+                      type: string
+                      pattern: '^([0-9A-Fa-f]){64}$'
+                  example: E3FC28889BDE31406465167F1D9D6A16DCA1FF67A3BABFA5E5A8596478848F78
+            responses:
+                200:
+                    description: The node information.
+                    content:
+                        application/json:
+                            schema:
+                                $ref: '#/components/schemas/NodeInfo'
+                404:
+                    description: There is no node with the given main account.
+                    content:
+                        application/json:
+                            schema:
+                                $ref: '#/components/schemas/Error'
+    /nodes/nodePublicKey/{nodePublicKey}:
+        get:
+            summary: Returns the node information for the given transport public key.
+            tags:
+                - node
+            operationId: getNodeByNodePublicKey
+            parameters:
+                - name: nodePublicKey
+                  required: true
+                  description: The node's transport public key.
+                  in: path
+                  schema:
+                      type: string
+                      pattern: '^([0-9A-Fa-f]){64}$'
+                  example: E3FC28889BDE31406465167F1D9D6A16DCA1FF67A3BABFA5E5A8596478848F78
+            responses:
+                200:
+                    description: The node information for the given transport public key.
+                    content:
+                        application/json:
+                            schema:
+                                $ref: '#/components/schemas/NodeInfo'
+                404:
+                    description: There is no node with the given transport public key.
+                    content:
+                        application/json:
+                            schema:
+                                $ref: '#/components/schemas/Error'
+    /nodesHostDetail:
+        get:
+            summary: Returns the list of hosting details for all the known nodes.
+            tags:
+                - node
+            operationId: getNodesHostDetail
+            responses:
+                200:
+                    description: The list of the hosting details.
+                    content:
+                        application/json:
+                            schema:
+                                type: array
+                                items:
+                                    $ref: '#/components/schemas/HostDetail'
+    /nodeStats:
```

Comment:
there are `/nodes/` `/nodes/nodePublicKey` and `/nodesHostDetails`

but this one and next one comes in singular `/nodeStats`, `/nodeHeightStats`

Reply:
Yes, you are right. Ideally it should be plural `/nodesStats` and `/nodesHeightStats`. This PR should address this https://github.com/symbol/statistics-service/pull/118.

Reply:
Regarding rename the `/nodeStats` and `/nodeHeightStats`.
It will break into the existing Explorer.

Can we release Explorer (without pagination enhancement)? @0x6861746366574